File: backend/src/main/java/com/hcg/simpletaskmanager/model/PageTaskModelList.java
package com.hcg.simpletaskmanager.model;

import java.io.Serializable;
import java.util.List;

public class PageTaskModelList implements Serializable {
    private long total;
    private List<TaskModel> taskModelList;

    public long getTotal() {
        return total;
    }

    public void setTotal(long total) {
        this.total = total;
    }

    public List<TaskModel> getTaskModelList() {
        return taskModelList;
    }

    public void setTaskModelList(List<TaskModel> taskModelList) {
        this.taskModelList = taskModelList;
    }
}

File: ui/src/model/PageTaskModelList.ts
import {TaskModel} from "@/model/TaskModel";

export class PageTaskModelList {
    total: number = 0;
    taskModelList: Array<TaskModel> = new Array<TaskModel>();

    constructor(total: number, taskModelList: Array<TaskModel>) {
        this.total = total;
        this.taskModelList = taskModelList;
    }
}
